src/views/Give/YieldRecipients.tsx

```diff
@@ -48,65 +52,67 @@ export default function YieldRecipients({ changeView }: RecipientModalProps) {
-        </Typography>
-        <TertiaryButton onClick={() => changeView(0)}>
-          <Trans>Donate to a cause</Trans>
-        </TertiaryButton>
+      <Box>
+        <Grid container spacing={2} justifyContent="center">
+          <Grid item xs={12} container justifyContent="center">
+            <Typography variant="body1">
+              <Trans>Looks like you haven’t made any donations yet</Trans>
+            </Typography>
+          </Grid>
+          <Grid item xs={12} container justifyContent="center">
+            <TertiaryButton onClick={() => changeView(0)}>
+              <Trans>Donate to a cause</Trans>
+            </TertiaryButton>
+          </Grid>
+        </Grid>
       </Box>
     );
   }
 
+  // TODO extract the table and styles into a common component
   return (
-    <Grid container item className="card-content">
-      <TableContainer>
-        <Table className="donation-table">
-          <TableHead>
-            <TableRow>
-              {!isSmallScreen && (
-                <TableCell align="left">
-                  <Typography variant="body1">
-                    <Trans>DATE</Trans>
-                  </Typography>
-                </TableCell>
-              )}
+    <TableContainer>
+      <Table className="donation-table">
+        <TableHead>
+          <TableRow>
+            {!isSmallScreen && (
               <TableCell align="left">
                 <Typography variant="body1">
-                  <Trans>RECIPIENT</Trans>
+                  <Trans>DATE</Trans>
                 </Typography>
               </TableCell>
-              {!isSmallScreen && (
-                <TableCell align="right">
-                  <Typography variant="body1">
-                    <Trans>DEPOSITED</Trans>
-                  </Typography>
-                </TableCell>
-              )}
+            )}
+            <TableCell align="left">
+              <Typography variant="body1">
+                <Trans>RECIPIENT</Trans>
+              </Typography>
+            </TableCell>
+            {!isSmallScreen && (
               <TableCell align="right">
                 <Typography variant="body1">
-                  <Trans>YIELD SENT</Trans>
+                  <Trans>DEPOSITED</Trans>
                 </Typography>
               </TableCell>
-              <TableCell align="right" className="manage-cell"></TableCell>
-            </TableRow>
-          </TableHead>
-          <Divider className="table-head-divider" />
-          <TableBody>
-            {isLoading ? (
-              <Skeleton />
-            ) : (
-              donationInfo.map(donation => {
-                return <DepositTableRow depositObject={donation} key={donation.recipient} />;
-              })
             )}
-          </TableBody>
-        </Table>
-      </TableContainer>
-    </Grid>
+            <TableCell align="right">
+              <Typography variant="body1">
+                <Trans>YIELD SENT</Trans>
```

Comment:
The alignment of the My Donations table is thrown off a bit on mobile? 
<img width="284" alt="image" src="https://user-images.githubusercontent.com/80423742/160191700-1fa32199-6d00-4ec4-80e1-a9713119de02.png">

Reply:
other than it looks good to go

Reply:
Thanks for spotting this! YieldRecipients was the last section that hadn't been converted to use the Grid component (it was using a Table and I was wary of messing with that), but this problematic behaviour pushed me to make that conversion and fix this issue. (Which it has done.) It now matches the design spec.

Large breakpoint:
<img width="1025" alt="Screenshot 2022-03-28 at 11 58 26 AM" src="https://user-images.githubusercontent.com/91760036/160354195-7f98e42d-8909-43b8-a3dc-b07edfa4cae9.png">

Medium:
<img width="739" alt="Screenshot 2022-03-28 at 11 58 44 AM" src="https://user-images.githubusercontent.com/91760036/160354227-15f567d8-3d12-4ff3-b142-b07f8150116b.png">

Extra-small:
<img width="500" alt="Screenshot 2022-03-28 at 12 01 28 PM" src="https://user-images.githubusercontent.com/91760036/160354294-3d45726f-6ce1-4efe-bb64-84d410afb562.png">

Extra-small (mobile):
<img width="299" alt="Screenshot 2022-03-28 at 12 09 34 PM" src="https://user-images.githubusercontent.com/91760036/160354630-78fa0e81-037a-4e15-a942-ea7cb782fc44.png">

Reply:
@appleseed-iii for your review

Reply:
thanks @0xJem . looks good